packages/hawtio/src/plugins/shared/operations/OperationForm.tsx

```diff
@@ -185,14 +182,57 @@ export const OperationForm: React.FunctionComponent<OperationFormProps> = props
     setIsDropdownOpen(!isDropdownOpen)
   }
 
+  const copyToClipboard = (text: string) => {
+    navigator.clipboard.writeText(text)
+    setCurrentAlerts(prev => [
+      ...prev,
+      <Alert key={`{alert-${prev.length + 1}`} isLiveRegion timeout={2000} title='Copied to clipboard' />,
```

Comment:
Default alert timeout we use throughout hawtio-next is 8s (= 8000). Unless it was the behaviour from the old Hawtio, we should apply the same timeout duration here as well.

Reply:
Alright. I checked the old-hawtio and 8 seconds seemed too long but I don't think there is any strong reason to not go with the default. It's patternfly's default as well so it'll make the code cleaner just removing that prop :smile: